src/components/multilineInput/styles/SplitMultilineInput.css

```diff
@@ -0,0 +1,22 @@
+.inputBlock {
```

Comment:
Is it possible to use sass instead of css? This way we could import variables from vapor when needed

Reply:
We lose a lot of the CSS Modules' power when using sass instead. But it is possible to import sass variables.

Reply:
Good to know!